scripts/taginfo_template.json

```diff
@@ -222,6 +222,13 @@
       "description": "Major roads under construction have a dotted line pattern and a more prominent color.",
       "doc_url": "https://openmaptiles.org/schema/#transportation"
     },
+    {
+      "key": "highway",
+      "value": "track",
+      "object_types": ["way"],
+      "description": "Track roads have a two-track line pattern that is dashed if unpaved and solid if paved.",
+      "doc_url": "https://openmaptiles.org/schema/#transportation"
+    },
```

Comment:
Also add an entry for `ford=yes`, as well as a second entry for `surface` that describes the stylistic variation on tracks.

Reply:
Thanks for the feedback. Added these taginfo entries in ab1e133.